docs/src/interface/operations.md

```diff
@@ -0,0 +1,17 @@
+
+# An interface for Lie group operations
+
+```@docs
+AbstractGroupOperation
+Identity
+```
+
+For some generic group operations, generic implementations are available.
```

Comment:
```suggestion
For some generic group operations, a generic implementation is available.
```
AFAIK the plural of "implementation" is rarely used.

Reply:
what would you prefer than for these generic function definitions?

Reply:
This comment doesn't refer to the code, only docstring and grammar.

Reply:
My question as well ;) I am really just not sure what to write instead.

Reply:
Rephrased it to “Some specific group operations allow to define functions for all manifolds they
do build a Lie group with. These are”

starting an overview afterwards, because this will list the general multiplication and the matrix multiplication case and which ever one we come up with generically.